tests/ieee802154_hal/readme.md

```diff
@@ -0,0 +1,32 @@
+# Used Devices
```

Comment:
I think it would be good to write here what the riotctrl related functions are doing. At least `txtspam`, `check_last_packet`, etc